app/src/fdroid/res/xml/preferences_global.xml

```diff
@@ -0,0 +1,310 @@
+<androidx.preference.PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why are you adding this file now? Was it missing from a previous commit?

Reply:
It's being added under the `fdroid` folder. So it is used for the fdroid build

Reply:
Yes. The question is why now and not before?